Doesn't build against node 7

## Issue or Feature
Ran into this doing the prebuilds. 2.2.0 doesn't build in Node 7 on any of the 3 platforms. Node 6 does. Should we add all versions to the build matrix?

## Steps to Reproduce
* `npm install` in the repo
* `src/CanvasRenderingContext2d.cc:1722:49: error: no member named 'InstanceOf' in 'v8::Object'`

Looks like it was caused by #1293 ([this line](https://github.com/Automattic/node-canvas/blob/master/src/CanvasRenderingContext2d.cc#L1722)). Should it be `> 7` instead?

## Your Environment
* Version of node-canvas (output of `npm list canvas` or `yarn list canvas`): 2.2.0
* Environment (e.g. node 4.2.0 on Mac OS X 10.8): any

--- a/src/CanvasRenderingContext2d.cc
+++ b/src/CanvasRenderingContext2d.cc
@@ -1718,7 +1718,7 @@ NAN_SETTER(Context2d::SetCurrentTransform) {
   Context2d *context = Nan::ObjectWrap::Unwrap<Context2d>(info.This());
   Local<Context> ctx = Nan::GetCurrentContext();
 
-#if NODE_MAJOR_VERSION > 6
+#if NODE_MAJOR_VERSION >= 8
   if (!Nan::To<Object>(value).ToLocalChecked()->InstanceOf(ctx, _DOMMatrix.Get(Isolate::GetCurrent())).ToChecked())
     return Nan::ThrowTypeError("Expected DOMMatrix");
 #endif

